### ISSUE TITLE
cannot bazel build freeze_graph on windows

### ISSUE THREAD

**Issue by eyaler** [tensorflow/tensorflow#12324]
using bazel 0.53, python 3.6.2 on windows 10 64 with tf-gpu 1.3.0rc2

bazel build tensorflow/python/tools:freeze_graph
.
ERROR: C:/users/eyaler/downloads/tensorflow-master/tensorflow-master/tensorflow/python/tools/BUILD:32:1: error loading package 'tensorflow/core': Encountered error while reading extension file 'protobuf.bzl': no such package '@protobuf_archive//': Traceback (most recent call last):
        File "C:/users/eyaler/downloads/tensorflow-master/tensorflow-master/tensorflow/workspace.bzl", line 119
                _apply_patch(repo_ctx, repo_ctx.attr.patch_file)
        File "C:/users/eyaler/downloads/tensorflow-master/tensorflow-master/tensorflow/workspace.bzl", line 110, in _apply_patch
                _execute_and_check_ret_code(repo_ctx, cmd)
        File "C:/users/eyaler/downloads/tensorflow-master/tensorflow-master/tensorflow/workspace.bzl", line 94, in _execute_and_check_ret_code
                fail("Non-zero return code({1}) when ...))
Non-zero return code(127) when executing 'c:\tools\msys64\usr\bin\bash.exe -c patch -p1 -d C:/users/eyaler/appdata/local/temp/_bazel_eyaler/trcps2te/external/protobuf_archive -i C:/users/eyaler/downloads/tensorflow-master/tensorflow-master/third_party/protobuf/add_noinlines.patch':
Stdout:
Stderr:       1 [main] patch (14524) C:\Program Files\Git\usr\bin\patch.exe: *** fatal error - cygheap base mismatch detected - 0x1802F3408/0x1802FD408.
This problem is probably due to using incompatible versions of the cygwin DLL.
Search for cygwin1.dll using the Windows Start->Find/Search facility
and delete all but the most recent version.  The most recent version *should*
reside in x:\cygwin\bin, where 'x' is the drive on which you have
installed the cygwin distribution.  Rebooting is also suggested if you
are unable to find another cygwin DLL.
 and referenced by '//tensorflow/python/tools:freeze_graph'.
ERROR: Analysis of target '//tensorflow/python/tools:freeze_graph' failed; build aborted.
INFO: Elapsed time: 85.042s

**Comment by reedwm** [comment]
This looks like a cygwin issue, not a TensorFlow issue. Try following the advice it provides (the sentence that starts with "Search for cygwin1.dll using..."), and asking on StackOverflow.

**Comment by eyaler** [comment]
thanks. i followed the advice but it was not relevant.

**Comment by drbaker** [comment]
For what it's worth, I also had this problem today while trying to build Tensorflow on Windows using Bazel.  I think the problem was that I had the msys2 shell installed alongside Git for Windows (which also installs a version -- possibly a newer version -- of the msys2 shell in the form of Git Bash).  After I uninstalled Git for Windows, this error went away.  

(That's not to say that my Bazel build started working... I've spent three days trying to get it to work and it still fails.  I'm basically to the point of giving up and going back to Ubuntu.)
